python/ray/autoscaler/_private/cli_logger.py

```diff
@@ -51,6 +51,14 @@ def disable(self):
             pass
 
         def __getattr__(self, name):
+            if name == "with_style":
+                @contextmanager
+                def wrapper(x):
+                    class identityClass:
+                        def __getattr__(self, name):
+                            return lambda y: y
+                    yield identityClass()
+                return wrapper
```

Comment:
Can you put this into a separate function, like:

```

@contextmanager
def with_style(self, style_name):
    class IdentityClass:
        ...
    yield IdentityClass()

def __getattr__(self, name):
    ...

Reply:
Sure!